internal/api/documents.go

```diff
@@ -413,6 +413,171 @@ Hermes
 			s.RenameFile(docID,
 				fmt.Sprintf("[%s] %s", docObj.GetDocNumber(), docObj.GetTitle()))
 
+			// Get document record from database so we can modify it for updating.
+			d := models.Document{
+				GoogleFileID: docID,
+			}
+			if err := d.Get(db); err != nil {
+				l.Error("error getting document from database",
+					"error", err,
+					"method", r.Method,
+					"path", r.URL.Path,
+					"doc_id", docID,
+				)
+				// Don't return an HTTP error because the database isn't the source of
```

Comment:
I was thinking that this would never change in this file - the new version of the API will be a different file and this file will just be deleted. I'll remember to return an HTTP error for v2 🙂